src/settings/ExpenseTypeSettings/ExpenseTypeSettings.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  injectIntl,
+} from 'react-intl';
+
+import { ControlledVocab } from '@folio/stripes/smart-components';
+
+import { EXPENSE_CLASSES_API } from '../../common/const';
+
+const hiddenFields = ['numberOfObjects', 'lastUpdated'];
+const visibleFields = ['name', 'code', 'externalAccountNumberExt'];
+const columnMapping = {
+  name: <FormattedMessage id="ui-finance.expenseType.name" />,
+  code: <FormattedMessage id="ui-finance.expenseType.code" />,
+  externalAccountNumberExt: <FormattedMessage id="ui-finance.expenseType.account" />,
+};
+
+class ExpenseTypeSettings extends Component {
+  constructor(props) {
+    super(props);
+    this.connectedControlledVocab = props.stripes.connect(ControlledVocab);
```

Comment:
have you tried to put it as `const ConnectedControlledVocab = stripesConnect(ControlledVocab);` and write component as functional?